src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ThemeServicePermissionTests.cs

```diff
@@ -225,6 +225,15 @@ public void DeleteTheme()
                 );
         }
 
+        private static Mock<IConfiguration> DefaultConfigurationMock()
```

Comment:
We actually already have a `MockUtils.CreateMockConfiguration()` method that you can use with no parameters.

Reply:
I get a failing test if I use this, as `CreateMockConfiguration` uses `MockBehavior.Strict`.

`Moq.MockException: IConfiguration.GetSection("enableThemeDeletion") invocation failed with mock behavior Strict.`

I think I may have tried this and switched to creating DefaultConfigurationMock instead as I think `enableThemeDeletion` isn't a `Tuple<string, string>` but a `Tuple<string, bool>` and I think that was causing me issues.

Reply:
Ahh fair enough!